Design_Document.md

```diff
@@ -60,78 +60,66 @@ Pages:
 * Upload: Allows users to upload images to the application.
 * Profile: Allows users to see their uploads, profile picture, bio, etc.
 
-Tables: 
+Tables:
 ```
-  User Table
+  Image Table
   --------------------------------
-  userId:String, partition key 
-  userName:String
+  userImage:String, partition key
+  dateTime:LocalDateTime
+  caption:String
 ```
 ```
   Comments Table
```

Comment:
Which of these "user" attributes refer to the image record and which indicate who made the comment?